pkg/common/utils.go

```diff
@@ -113,3 +114,19 @@ func ConvertSITimeout(millis int64) time.Duration {
 	}
 	return time.Duration(result)
 }
+
+func GetIgnoreUnschedulable(tag map[string]string) bool {
+	var ignore bool
+	var err error
+	if ignoreUnschedulable, ok := tag[interfaceCommon.DomainYuniKorn+interfaceCommon.KeyIgnoreUnschedulable]; !ok {
+		// if there isn't ignoreUnschedulable tag, set it to false
+		ignore = false
+	} else {
+		ignore, err = strconv.ParseBool(ignoreUnschedulable)
+		if err != nil {
+			log.Logger().Warn("Failed to convert allocationTag ignoreUnschedulable from string to bool")
+			ignore = false
+		}
+	}
+	return ignore
+}
```

Comment:
this can be simplified to: 

```
if ignoreUnschedulable, ok := tag[xxx]; ok {
  // tag found
 if  ignore, err = strconv.ParseBool(ignoreUnschedulable); err == nil {
    // parse successful
    return ignore
  }
}

// every other cases, return false (default)
return false
```

Reply:
Sure, done.